app/views/specialist_document/show.html.erb

```diff
@@ -10,7 +10,13 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds responsive-top-margin">
-    <%= render "govuk_publishing_components/components/title", @content_item.title_and_context %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: content_item.title,
+      average_title_length: "long",
```

Comment:
I think you still want to remove this? (this is valid for the title component, not for the heading one)

Reply:
Good spot! Thanks. I copy and pasted from the case study view 🙈 , which will also need to be updated (in a separate change)

Reply:
I've made the change and fixed up.